tools/tertiary-analysis/dropletutils/dropletutils_macros.xml

```diff
@@ -5,7 +5,8 @@
 
   <xml name="requirements">
     <requirements>
-      <requirement type="package" version="0.0.1">dropletutils-scripts</requirement>
+      <requirement type="package">openblas</requirement>
```

Comment:
Please remove this if you have added the dependency at the conda package, thanks!

Reply:
Already done, just waiting for the new build to be available before I push :-)